### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by ViktorVoitovich**

### CODE REVIEW

File: babel.config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (AliakseiBychyk) [inline babel.config.json:1]: Please rename file into `.babelrc` , such convention is used for configuration json files.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "nodejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "task1": "nodemon --exec babel-node src/task1.js",
+    "task2": "nodemon --exec babel-node src/task2.js"
```
Comment (AliakseiBychyk) [inline package.json:8]: In order to get better evaluation, please implement additionally the same functionality not using babel at all. For example you may have such scripts:
`"task1": "nodemon src/task1.js"`
`"task1:babel": "nodemon -e babel-node src/task1babel.js"`

File: src/task2.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { pipeline } from 'stream';
+import csvtojson from 'csvtojson';
+import fs from 'fs';
+
+const csvFilePath = 'src/csv/test-data.csv';
+const outputFilePath = 'result.txt';
+
+pipeline(
+    fs.createReadStream(csvFilePath),
```
Comment (AliakseiBychyk) [inline src/task2.js:9]: Please, implement that functionality also using `readFile` `writeFile` methods in the separate file and create separate script in the `package.json`

**Review by AliakseiBychyk — CHANGES_REQUESTED** [review]

File: babel.config.json (hunk 1 above)
Comment (ViktorVoitovich) [reply babel.config.json]: @AliakseiBychyk done

File: package.json (hunk 2 above)
Comment (ViktorVoitovich) [reply package.json]: @AliakseiBychyk done

File: src/task2.js (hunk 3 above)
Comment (ViktorVoitovich) [reply src/task2.js:9]: @AliakseiBychyk done

**Review by AliakseiBychyk — APPROVED** [review]

**Pull request merged**